src/git.ts

```diff
@@ -103,7 +102,7 @@ export async function deploy(): Promise<string> {
       If the branch doesn't exist it gets created here as an orphan.
     */
   const branchExists = await execute(
-    `git ls-remote --heads ${repositoryPath} ${action.branch} | wc -l`,
+    `git ls-remote --heads deployment-repo ${action.branch} | wc -l`,
```

Comment:
As `deployment-repo` is used in a number of different locations across different functions, I think a good improvement would be to make it a variable that gets imported from the `constants` file. That way if it's changed in one place it's changed in all. I don't think a user will ever want to modify this so let's keep it out of the configuration object.